explorer/templates/explorer/preview_pane.html

```diff
@@ -156,21 +156,21 @@ <h3>{{ snapshots|length }} Snapshots <small>(oldest first)</small></h3>
                             </div>
                         {% else %}
                             <div style="margin: 6em;">
-                                <p>{% blocktrans %}
+                                <p>{% blocktranslate %}
```

Comment:
```suggestion
                                <p>{% blocktranslate trimmed %}
```

Trimmed on these will help in the translation files.